### ISSUE TITLE
invisible characters cause all sorts of incorrect errors

### ISSUE THREAD

**Issue by ctcpip** [ember-template-lint/ember-cli-template-lint#293]
when (invisible) ASCII control characters exist in a template, all sorts of false linting errors are reported.  in this case, the chars are ASCII dec 14 (shift out).  I'm not sure what the best solution is for this.  add errors for these characters and/or ignore them for linting?

example:

```
<div>
  <div>
    <div>

    </div>
  </div>
  <div>
    <div>

    </div>
    <div>

    </div>
  </div>
</div>

```

produces the following output:

```
  1:5  error  Non-translated string used  bare-strings
  6:8  error  Non-translated string used  bare-strings
  1:5  error  Incorrect indentation for `
  ` beginning at L1:C5. Expected `
  ` to be at an indentation of 2 but was found at 0.  block-indentation
  6:8  error  Incorrect indentation for `div` beginning at L2:C3. Expected `</div>` ending at L6:C8 to be at an indentation of 3 but was found at 2.  block-indentation
  3:4  error  Incorrect indentation for `<div>` beginning at L3:C4. Expected `<div>` to be at an indentation of 5 but was found at 4.  block-indentation
  14:8  error  Incorrect indentation for `div` beginning at L7:C5. Expected `</div>` ending at L14:C8 to be at an indentation of 5 but was found at 2.  block-indentation
  8:4  error  Incorrect indentation for `<div>` beginning at L8:C4. Expected `<div>` to be at an indentation of 7 but was found at 4.  block-indentation
  11:4  error  Incorrect indentation for `<div>` beginning at L11:C4. Expected `<div>` to be at an indentation of 7 but was found at 4.  block-indentation
```

**Comment by Turbo87** [comment]
@ctcpip can you move this issue to https://github.com/ember-template-lint/ember-template-lint?

**Comment by ctcpip** [comment]
closing in favor of https://github.com/ember-template-lint/ember-template-lint/issues/387
